docs/compute-resources.md

```diff
@@ -217,6 +320,113 @@ spec:
 Here, the minimum of the "max" values is the output "max" value, and likewise for "default" and "defaultRequest".
 The maximum of the "min" values is the output "min" value.
 
+### Configure Task-level Compute Resources on TaskRun with LimitRange
+
+**([alpha only](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features))**
+
+User can use `LimitRange` to configure min or max compute resources for a `Task` or `Step`.
+
+(Note: The Task-level compute resources configuration is an alpha feature at this moment.)
+
+Here are examples with a task-level resource requirements:
+
+e.g.
+
+Applying min:
+
+```yaml
+kind: LimitRange
+spec:
+  limits:
+    - min:
+        cpu: 200m
+---
+kind: Task
+spec:
+  steps:
+    - name: step1 # applied with min
+    - name: step2 # applied with min
+    - name: step3 # not applied
+---
+kind: TaskRun
+spec:
+  computeResources:
+    requests:
+      cpu: 1
+```
+
+| Step name | CPU request |
+| --------- | ----------- |
+| step1     | 800m        |
+| step2     | 200m        |
+| step3     | N/A         |
+
```

Comment:
I'd prefer not to include examples that state what the resource requirements on containers will end up being, since this is more of an implementation detail.

Reply:
was thinking if this could help users understand how their configure will take an effect, so better know how to configure their requirements. how do you think about it?

Reply:
hm, on second thought it might be worth including an example-- my main concern is that there are multiple valid ways to implement task-level resource requirements, so I don't necessarily want to tie ourselves to one way in the documentation. but I'll leave this up to you.

Reply:
this is a good point as the updating logic is not completed now and could be adjusted when going into the details. 

I backed these changes. let me remove them now, and see if I can offer one meaningful example after impled the resource config/update logic.